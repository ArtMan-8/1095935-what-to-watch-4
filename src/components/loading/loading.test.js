import React from 'react';
import renderer from 'react-test-renderer';

import Loading from './loading.jsx';

describe(`Loading`, () => {
  it(`Render Loading`, () => {
    const tree = renderer.create(
        <Loading />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
