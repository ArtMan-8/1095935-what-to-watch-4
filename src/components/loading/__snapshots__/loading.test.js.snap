// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Render Loading 1`] = `
<div
  className="sweet-loading"
>
  <div
    className="css-13xsomd-override"
  >
    <div
      className="css-qr5dvv"
    />
    <div
      className="css-1hedhch"
    />
  </div>
</div>
`;
